src/main/services/downloader.ts

```diff
@@ -140,6 +140,9 @@ export class Downloader {
                 method: "GET",
                 encoding: undefined,
                 headers,
+                agentOptions: {
+                    rejectUnauthorized: false,
```

Comment:
Why are we seeing this problem now, and not before? Has something changed in our code that triggers this new behaviour? Or is that a change in the HTTP servers?

Reply:
This should really only be used in development mode, not for production (security hole with man-in-the-middle attacks, if we break the certificate validation chain).

Reply:
Answering my own question:
https://www.sslshopper.com/ssl-checker.html#hostname=https://lcp-prod.edrlab.org
"One of the root or intermediate certificates has expired (10 days ago)."